bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/VolumeControlVolume.java

```diff
@@ -44,6 +45,10 @@
     @Override
     public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command command) {
         final PercentType percent;
+        if (RefreshType.REFRESH == command) {
+            handler.getSocket().getVolume(createResponseListener(channelId, handler));
+            return;
```

Comment:
Maybe suppress this return and add else jus after like you do in the class MediaControlPlayer or VolumeControlMute ?

Reply:
what follows is not a complete if else cascade.  I find it more readable to jump out early